src/main/java/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,68 @@
+package controller;
+
+import dao.BetDao;
+import dao.BetDaoImpl;
+import dao.UserDaoImpl;
+import java.util.Scanner;
+import model.Bet;
+import model.User;
+import services.UserService;
+
+public class ConsoleHandler {
+    private static final int DATA_QUANTITY = 2;
+    BetDao betDao = new BetDaoImpl();
```

Comment:
let's keep userDao here as well and add access modifier

Reply:
done